packages/angular/stories/generated/dialog-service.mdx

```diff
@@ -0,0 +1,151 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Components/Dialog Service" />
+
+# Components/Dialog Service
+
+{/*
+This file is automatically generated from dialog-service.stories.ts
+Do not edit this file directly. Run 'yarn extract-templates' to regenerate.
+To add custom introduction content, create dialog-service.intro.md */}
+
+## Dialog Service Example
+
+A simple example of using the DialogService to open a dialog. This works with any dialog. How you can create a dialog can be seen in the next example.<br/><br/>Note the <code>@Inject(DIALOG_DATA)</code> that is used to get the data passed to the dialog component.
+
+
+```typescript
+import { ButtonComponent } from '../src/button/button.component';
+import { DialogComponent } from '../src/dialog/dialog.component';
+import { FormFieldComponent } from '../src/form/form-field.component';
+import { FormComponent } from '../src/form/form.component';
+import { TextAreaDirective } from '../src/forms/text-area.directive';
+import { TextFieldDirective } from '../src/forms/text-field.directive';
+import { DialogRef, DialogService } from '../src/services/dialog.service';
+import { TextAreaComponent } from '../src/text-area/text-area.component';
+import { TextFieldComponent } from '../src/text-field/text-field.component';
```

Comment:
It would be helpful to show users where they should import these components and directives from, for example:

```
import { CheckboxDirective, CheckboxGroupDirective, RadioGroupDirective, SelectDirective, TextFieldDirective } from '@sl-design-system/angular/forms';
import { TextFieldComponent } from '@sl-design-system/angular/text-field';
import { SelectComponent } from '@sl-design-system/angular/select';
...
```